### DISCUSSION TITLE
I would like to support for Godot's GDScript.

### DISCUSSION THREAD

**Discussion by DragonParts** [massCodeIO/massCode#94] [Ideas]
I routinely create games with Godot, which comes with its own script, GDScript, similar to Python.

Please support GDScript as well.

**Comment by mowfax** [comment]
Since Godot is getting more popular by the day, the addition of GDScript would be very helpful.

**Comment by ghutchi1** [comment]
Yes please, gdscript would be very helpful.

**Comment by ghutchi1** [comment]
I worked out that as a sort of work around, you can change it to Python, this is fairly close, it uses same tab stops.

**Comment by mowfax** [comment]
did the switch to massCode v4 make support for new languages more realistic?
Also, there is a textmate syntax file in the official repository (MIT license):
https://github.com/godotengine/godot-vscode-plugin/tree/master/syntaxes
